internal/events/eventstream.go

```diff
@@ -34,14 +33,15 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-type DistributionMode string
-
-const (
-	DistributionModeBroadcast DistributionMode = "broadcast"
-	DistributionModeWLD       DistributionMode = "workloadDistribution"
-)
-
 const (
+	// when multiple websocket client are connected for the same topic, send events to all connected clients
+	DistributionModeBroadcast = "broadcast"
+	// when multiple websocket client are connected for the same topic, send events to only one of the connected clients
```

Comment:
Maybe `send each event to only one of the connected clients` (rather than implying the same client will get all events)